cosmwasm/packages/wasmi-runtime/src/registration/offchain.rs

```diff
@@ -211,3 +222,84 @@ pub unsafe extern "C" fn ecall_key_gen(
     trace!("ecall_key_gen key pk: {:?}", public_key.to_vec());
     sgx_status_t::SGX_SUCCESS
 }
+
+#[cfg(not(feature = "SGX_MODE_HW"))]
+fn verify_local_report(_cert: &[u8]) -> sgx_status_t {
+    sgx_status_t::SGX_SUCCESS
+}
+
+#[cfg(feature = "SGX_MODE_HW")]
+fn verify_local_report(cert: &[u8]) -> sgx_status_t {
```

Comment:
I prefer returning `SgxResult` if its not an extern function

Reply:
I'm just removing the return code it's unused anyway